### PR TITLE
Updating the Blog section of BG.software

### PR DESCRIPTION
**Pull request by johnscottBG**
I'm a little out of practice but I don't think I messed anything up :)
Can you review please?
THANKS.

### CODE REVIEW

File: _posts/2020-07-23-what-was-one-crisis-is-now-policy.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+author : John Scott
+
+comments : true
+
+tags : [John Scott, Software,COVID-19]
+---
+
+<p>For most of us, we're at week 18 of this global health battle. It feels like it's been forever.</p>
+<p>We likely initially panicked, then furloughed/laid off people. We scrubbed everything in sight. Some of got a mask. Some of us didn't. We cut back, slimmed down, or maybe even staffed up. My wife works for an EdTech firm, deploying distance learning software solutions for K-12. She can barely keep up.</p>
```
Comment (aajungroth) [inline _posts/2020-07-23-what-was-one-crisis-is-now-policy.md:17]: "Some of got a mask." should be "Some of us got a mask."

File: _posts/2020-07-23-what-was-one-crisis-is-now-policy.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+<p>Businesses are scrambling, spending precious capital to adapt, and hoping for some stability. The one thing we don't have much of are things reliable and expected and constant. Not just in my home state but across the country, in your county and town.</p>
+<p>You made changes to your customer approach, your business model, and perhaps even your technology. One thing I noticed right away - the touchless menu. My friends own a restaurant. They didn't want to deal with menus and all those hands on them, so they employ a QR code a diner scans when they arrive. Up comes the menu and drinks/wine. A masked server takes our ouder, then delivers our meal to the table. We pay through an app and we're out. The restaurant has a safe flow, proper spacing too. We feel safe there and of course we want to support them and other local small businesses. </p>
```
Comment (aajungroth) [inline _posts/2020-07-23-what-was-one-crisis-is-now-policy.md:21]: I think "so they employ a QR code a diner scans" should be "so they employ a QR code that a diner scans".

File: _posts/2020-07-23-what-was-one-crisis-is-now-policy.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+<p>We've learned we <em>really</em> don't need to fly anywhere to have a meeting. It's hard to imagine right now what airlines are going to do to adapt.  Zoom has zoomed, trading in the 200+ dollar-per-share range, up from 75 dollars and change in Q1 of this year, security issues be damned.</p>
+<p>24 Hour Fitness was devastated by the lockdowns. They're bankrupt.</p>
+<p>With every story of distress, we're reminded that folks who make vodka now also make hand sanitizer. Ford shifted from all cars to make ventilators. Fashion houses are selling fancy masks. The plexiglas industry certainly doesn't appear to be hurting, now that we have found a new use for their product (the spit shield). And the "six feet apart" sticker companies, same thing.</p>
+<p>You understand the need for adaptation in running a business. Staying nimble means staying on top of what's happening in the marketplace. A reordering of priorities is happening. A shuffling of processes is underway. Try and look at the whole chess board ( even if you don't play)! See as much as you can. You'll make much smarter moves that way.</p>
```
Comment (aajungroth) [inline _posts/2020-07-23-what-was-one-crisis-is-now-policy.md:27]: "( even if you don't play)" should be "(even if you don't play)".

File: _posts/2020-07-23-what-was-one-crisis-is-now-policy.md (hunk 1 above)
Comment (MHealton) [reply _posts/2020-07-23-what-was-one-crisis-is-now-policy.md:17]: I noticed a couple of little tweaks needed:
- you'll need to add '---' to the top of this file to get the page to render your image and fill in all the variables you entered.
- the word adlib should be hyphenated if it's being used as a verb or adjective: 'ad-lib'
- the name of the file says '...one-crisis...' but I think it should say '...once-crisis...'

**Pull request merged**